### PR TITLE
Updated General Styling of FAQ Page

### PR DESCRIPTION
**Pull request by laurasofi1024**
Added all faq boxes and background color.

Changed margin size of faq boxes. 

Pending: Updates on how to implement the special styling for the background/navbar

### CODE REVIEW

File: src/containers/FAQ.js [hunk 1]
```diff
@@ -1,23 +1,60 @@
 import React from 'react';
 import { FaqBox } from '../components/faqbox';
+import styled from 'styled-components';
+
+const PageWrapper = styled.main`
+  background-color: #E5E5E5;
```
Comment (erin2722) [inline src/containers/FAQ.js:6]: Once you merge in master, there is a theme provider for styled components and you can add this color in theme.js and change this to (with the color name replaced):
```CSS
color: ${props=>props.theme.colors.red};
```

File: src/containers/FAQ.js [hunk 2]
```diff
@@ -1,23 +1,60 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const Title = styled.h1`
+    margin-top: 100px;
```
Comment (erin2722) [inline src/containers/FAQ.js:13]: Change both of these to rem to improve adjustment to different screen sizes

**Review by erin2722 — CHANGES_REQUESTED** [review]
Looks great 👍  Once you merge in master, add in the theme provider, and make a few small changes it is good to go!

File: src/containers/FAQ.js (hunk 1 above)
Comment (laurasofi1024) [reply src/containers/FAQ.js:6]: Sound great! I will merge and add the theme provider

**Pull request merged**